Ja translation 23.0.0.11

dmuelle opened the pull request:
post on draft: https://draft-openlibertyio.mqj6zf7jocq.us-south.codeengine.appdomain.cloud/ja/blog/2023/11/14/23.0.0.11.html

una-tapa reviewed (commented):
In the second paragraph, I needed to add a space after featureUtility` to make it appear correctly when rendered.  Will you help update it please?  Otherwise, it looks great. Thank you, David. 

Open Liberty の今回のリリースでは、`featureUtility`コマンドもアップデートされました。このコマンドは、Libertyに新しい機能をインストールするときに、デフォルトでフィーチャーの署名を検証するようになりました。

dmuelle commented on posts/ja/2023-10-17-23.0.0.10.adoc:
```diff
@@ -27,7 +27,7 @@ Laura Cowen <https://github.com/lauracowen>
 Java 21がついに登場しました！ Java 21は、Java 17がリリースされてから初めての長期サポート（LTS）リリースです！新機能や変更点がいくつか提供されるので、ぜひチェックしてください。特に、仮想スレッドが導入されました。
 
-Open Liberty の今回のリリースでは、`featureUtility`コマンドもアップデートされました。このコマンドは、Libertyに新しい機能をインストールするときに、デフォルトでフィーチャーの署名を検証するようになりました。
+Open Liberty の今回のリリースでは、`featureUtility` コマンドもアップデートされました。このコマンドは、Libertyに新しい機能をインストールするときに、デフォルトでフィーチャーの署名を検証するようになりました。
```
@una-tapa - here's the fix for that typo

dmuelle commented:
Apologies @una-tapa - I had accidentally linked to the wrong translated post (23.0.0.10) and the typo you found was there. I've fixed the typo and also updated the link in my previous comment to point to the 23.0.0.11 post. Would you re-review when you have a chance?

una-tapa reviewed (approved):
Thank you, David!
